script.js

```diff
@@ -58,62 +59,87 @@
     return uniq([...pluralized, ...possibleAliases])
   }
 
-  const enrichResources = (resources) =>
+   enrichResources = (resources) => (
     resources.map((a) =>
       ({ ...a, extraTags: addTagAliases(a.tags) }))
+  )
 
   const buildResourceItem = ({ href, title, tags, description }) => {
     const link = `<a href="${href}" rel="noopener noreferrer nofollower" target="_blank">${title}</a/>`
     const tagList = `<small>tags: ${tags.join(' ')}</small>`
     return `<li>${[link, description, tagList].filter(Boolean).join('<br>')}</li>`
   }
 
+  const fetchy = (resources) => {
+    // enrich resource with plurals and tag aliases from functions above
+    const enrichedResources = enrichResources(resList)
+
+    const fuse = new window.Fuse(enrichResources(resList), fuseOptions)
+
+    // set up filtering code
+    const addListFilter = () => {
+      const ul = $(document.body).find('ul')
+
+      // actual filtering function
+      const filterRows = (e) => {
+        //console.log("I wanna show all of it!")
+  
+        var completeList = itemsList.concat(items)
+        //console.log(completeList)
+
+        $('ul').html(completeList)
+        addListFilter()
```

Comment:
Probably because this is a draft, but calling `addListFilter` inside `addListFilter` won't be good.